### PR TITLE
[docs] Clarify feature split between pro and premium

### PR DESCRIPTION
**Pull request by oliviertassinari**
A proposal for the feature split between the Pro $249/dev (eq. $20/month) and Premium $599/dev (eq. $50/month) plans.

A preview: https://deploy-preview-779--material-ui-x.netlify.app/components/data-grid/getting-started/#feature-comparison I have used the same icons as the ones we have on the coming pricing page.

Capture d’écran 2021-01-06 à 19 42 05

### CODE REVIEW

**Comment by oliviertassinari** [comment]
@mui-org/x  I have reworked the content, please review.

**Comment by DanailH** [comment]
I'm ok with this split. From what I can see the difference is that these features

- Range selection
- Excel export
- Clipboard
- Grouping
- Aggregation
- Pivoting

will be moved to the 'Premium' license and the rest will remain the same but from a business point of view, we will call them 'Pro' instead of 'Enterprise'. Am I understanding this correctly?

File: docs/src/pages/components/data-grid/group-pivot/group-pivot.md [hunk 1]
```diff
@@ -7,39 +7,39 @@ components: DataGrid, XGrid
 
 <p class="description">Use grouping, pivoting and more to analyse the data in depth.</p>
 
-## 🚧 Tree data [<span role="img" title="Enterprise">⚡️</span>](https://material-ui.com/store/items/material-ui-x/)
+## 🚧 Tree data [<span class="pro"></span>](https://material-ui.com/store/items/material-ui-x/)
```
Comment (DanailH) [inline docs/src/pages/components/data-grid/group-pivot/group-pivot.md:10]: Just a note, maybe it is just me but I found it a bit difficult to differentiate the Pro from the Premium icon as the only difference is the color (and they are both blue, just different shades of it).

**Review by DanailH — APPROVED** [review]

File: docs/src/pages/components/data-grid/group-pivot/group-pivot.md (hunk 1 above)
Comment (oliviertassinari) [reply docs/src/pages/components/data-grid/group-pivot/group-pivot.md:10]: The shares are also different, premium is longer.

**Comment by oliviertassinari** [comment]
> will be moved to the 'Premium' license and the rest will remain the same but from a business point of view, we will call them 'Pro' instead of 'Enterprise'. Am I understanding this correctly?

Regarding the terminology, I think that:

- DataGrid, is what we reference as the Community version, with an MIT license.
- XGrid, is what we reference as Enterprise, with a commercial license. We will probably add a GPLv3 (open source license) along the road.
- DataGrid & XGrid are part of the Material-UI X product. We might move the DatePicker, Autocomplete, and TreeView to Material-UI X, independently from the license. Material-UI X focuses on the hard and richer components, MIT or Commercial licensed. Material-UI focuses on building a design system per se.
- Pro and Premium are two different plans of the Enterprise version, which includes the Material-UI X product.

**Comment by dtassone** [comment]
I think the clipboard export should be in the pro as well as it's a basic feature and it is already working in the current version

**Comment by oliviertassinari** [comment]
> I think the clipboard export should be in the pro as well as it's a basic feature and it is already working in the current version

@dtassone OK, what I had in mind is already covered by the "Range selection" feature inside the premium plan.

**Comment by mbrookes** [comment]
Jost to add that, if there's any uncertainty, we can make premium features pro in the future, but much harder to go the other direction.

**Comment by oliviertassinari** [comment]
> If there's any uncertainty, we can make premium features pro in the future, but much harder to go the other direction.

@dtassone What's the main reason for customers to purchase ag-Grid? Are these reasons under the "Premium" plan? If not, we should probably reconsider. The "Pro" plan is more meant for any professional developers.

**Comment by dtassone** [comment]
Should we summarise the differences as
Pro => for an advanced data table and a nice way of displaying data
Premium => For data analysis, and more advanced use cases

**Review by dtassone — APPROVED** [review]

**Pull request merged**